### PR TITLE
Deprecrate identity-proofer-gem too

### PR DESCRIPTION
**Pull request by zachmargolis**
See https://github.com/18F/identity-idp/pull/5142

### CODE REVIEW

File: _articles/github.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ View a [list of all repositories](https://github.com/topics/login-gov) tagged fo
 - [**18F/omniauth_login_dot_gov**](https://github.com/18F/omniauth_login_dot_gov)
   Provides an Omniauth strategy for login.gov that helps other projects integrate with login.gov more easily. Some use cases include the [identity-dashboard](https://github.com/18f/identity-dashboard) as well as Touchpoints, USMC and search.gov
 
-- [**18f/identity-proofer-gem**](https://github.com/18f/identity-proofer-gem)
-  Provdes a generic interface around identity proofing (IDV, identity verification) that is implemented by other gems
+- [**18f/identity-proofer-gem**](https://github.com/18f/identity-proofer-gem){: .deprecated-link} (Archived)
```
Comment (aduth) [inline _articles/github.md:41]: In #127 you moved the repositories to the bottom of the list. Do we want to move this one down below `18f/identity-validations`?

**Review by aduth — APPROVED** [review]
👍

File: _articles/github.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/github.md:41]: good point, I forgot I did that 🤦 thanks

**Review by solipet — APPROVED** [review]
Thanks!

**Pull request merged**